docs/reference/search/request/scroll.asciidoc

```diff
@@ -167,7 +167,7 @@ clear a search context manually with the `clear-scroll` API:
 curl -XDELETE localhost:9200/_search/scroll \
      -d 'c2Nhbjs2OzM0NDg1ODpzRlBLc0FXNlNyNm5JWUc1' <1>
 ---------------------------------------
-<1> The `scroll_id` can be passed in the request body or in the query string.
+<1> The `_scroll_id` can be passed in the request body or in the query string.
```

Comment:
Should probably use the same phrasing as in the mention of scroll_id in the query string, because of the mix of `scroll_id` and `_scroll_id`